torch/nn/parallel/distributed.py

```diff
@@ -1063,9 +1064,6 @@ def register_comm_hook(self, state: object, hook: callable):
             ``register_comm_hook`` API, but it is subject to some subtle differences compared
             to ``torch.futures.Future``.
 
-        .. warning ::
-            DDP communication hook is experimental and subject to change.
-
```

Comment:
Why was this warning removed? We apply the "experimental and subject to change" warning to all "beta" features and I believe ddp comm hook is still "beta" right?

Reply:
I see. Recovered.